host_source.go

```diff
@@ -402,10 +402,10 @@ func (h *HostInfo) HostnameAndPort() string {
 }
 
 func (h *HostInfo) ConnectAddressAndPort() string {
-        h.mu.Lock()
-        defer h.mu.Unlock()
-        addr, _ := h.connectAddressLocked()
-        return net.JoinHostPort(addr.String(), strconv.Itoa(h.port))
+	h.mu.Lock()
```

Comment:
I don't think it makes sense to format files which were not touched by the fix.

Reply:
![image](https://github.com/gocql/gocql/assets/81249826/a7a9f2d4-25cb-4348-b71b-d5a613a8de14)

Reverted these changes, but it seems that they stayed inside this PR :\